README.md

```diff
@@ -54,21 +72,65 @@ migration_service.run_migrations(direction="downgrade")
 migration_service.generate_migration_script(migration_name="example_migration")
 </pre>
 
-## File Structure
+## Configuration
+
+This package allows you to customize the location where migration scripts are stored. By default, the migration scripts will be stored in a directory called `examples` within your project.
+
+### Changing the Migration Scripts Directory
+
+You can change the location of the migration scripts directory by setting the `MIGRATION_SCRIPTS_DIR` environment variable before running your application or executing any scripts from this package.
+
+#### Unix-based Systems (Linux, macOS)
 
-your_project/
-├── migrations/
-│   ├── migration_resource.py
-│   ├── migration_utils.py
-│   └── versions/
-│       ├── 0001_initial_migration.py
-│       └── 0002_add_field_to_table.py
+On Unix-based systems, you can set the `MIGRATION_SCRIPTS_DIR` environment variable like this:
+   `export MIGRATION_SCRIPTS_DIR="/path/to/custom/migration_scripts_directory"`
+Or in the corresponding Dockerfile of your service by specifying MIGRATION_SCRIPTS_DIR 
+   `ENV MIGRATION_SCRIPTS_DIR=$SCRIPT_DIR`
```

Comment:
I **think** you intended these to be separate lines, which needs slightly different markdown

```suggestion
On Unix-based systems, you can set the `MIGRATION_SCRIPTS_DIR` environment variable like this:

    export MIGRATION_SCRIPTS_DIR="/path/to/custom/migration_scripts_directory"

Or in the corresponding Dockerfile of your service by specifying MIGRATION_SCRIPTS_DIR 

    ENV MIGRATION_SCRIPTS_DIR=$SCRIPT_DIR
```